client/src/pages/LibrarySearchPage/LibrarySearchPage.js

```diff
@@ -75,6 +75,30 @@ class LibrarySearchPage extends Component {
       });
   };
 
+  fetchTableDataByType = type => {
+    axios
+      .get(`/v1/library/${type}`)
+      .then(response => {
+        let pageUrls = this.state.pageUrls;
+        if (typeof response.data.nextPage !== 'undefined') {
```

Comment:
You can remove this if block since the  API endpoints you created are not returning `nextPage`.